From: Philippe Coval <rzr@users.sf.net>
Date: Thu, 20 Jun 2019 10:43:04 +0200
Subject: build: Install headers

Reintroduce this feature after 99ec3710

It was useful for debian packaging (iotjs-dev),
to match quality standards.

Email has been updated for community support

Relate-to: https://packages.qa.debian.org/i/iotjs.html
Fowarded: https://github.com/jerryscript-project/iotjs/pull/1896
Relate-to: https://github.com/Samsung/iotjs/pull/1145
Change-Id: Idd9a18bbe692baeb468152df5149374512e9350b
Bug: https://github.com/jerryscript-project/iotjs/issues/1945
IoT.js-DCO-1.0-Signed-off-by: Philippe Coval rzr@users.sf.net
---
 cmake/iotjs.cmake | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/cmake/iotjs.cmake b/cmake/iotjs.cmake
index b989b6d..1b58179 100644
--- a/cmake/iotjs.cmake
+++ b/cmake/iotjs.cmake
@@ -495,6 +495,12 @@ set(IOTJS_PUBLIC_HEADERS
   ${LIBUV_HEADERS}
 )
 
+# Install headers
+if("${INCLUDE_INSTALL_DIR}" STREQUAL "")
+  set(INCLUDE_INSTALL_DIR "include/iotjs")
+endif()
+install(FILES ${IOTJS_PUBLIC_HEADERS} DESTINATION ${INCLUDE_INSTALL_DIR})
+
 # Configure the libiotjs
 set(TARGET_LIB_IOTJS libiotjs)
 if(CREATE_SHARED_LIB)
